plugins/osdn/netid/vnid/vnid.go

```diff
@@ -0,0 +1,80 @@
+package vnid
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/openshift/openshift-sdn/pkg/netid"
+)
+
+type VNIDRange struct {
+	Base uint
+	Size uint
+}
+
+// Contains tests whether a given vnid falls within the Range.
+func (r *VNIDRange) Contains(vnid uint) (bool, uint) {
```

Comment:
Could you add something to the code doc about what 'offset' is and why it gets used on return, instead of using the passed in vnid directly?